src/sw/controller.js

```diff
@@ -75,6 +70,7 @@ function getRetrievalClientId () {
 function getClientKey() {
     const urlObj = new URL(self.location.href)
     const clientKey = urlObj.searchParams.get('clientKey')
+    console.log('cleint key', clientKey)
```

Comment:
stop adding console.log to prod code >.<

if you want to log stuff conditionally, feel free to use the `debug` package which is already installed.